### PR TITLE
Fix wrong type error in admission

### PR DESCRIPTION
**Pull request by qiujian16**
Signed-off-by: Jian Qiu <jqiu@redhat.com>

### CODE REVIEW

**Review by morvencao — COMMENTED** [review]
/LGTM

File: controlplane/hack/lib/util.sh [hunk 1]
```diff
@@ -283,13 +283,13 @@ EOF
 function kube::util::write_client_kubeconfig {
     local sudo=$1
     local dest_dir=$2
-    local ca=`cat $3 | base64 -w0`
+    local ca=`cat ${dest_dir}/$3 | base64`
```
Comment (morvencao) [inline controlplane/hack/lib/util.sh:286]: is this intent? removing `-w0`?

Comment (clyang82) [inline controlplane/hack/lib/util.sh:286]: why remove `-w0`? I think we need to disable the line wrapping.

**Pull request merged**